lms/djangoapps/discussion/templates/discussion/edx_ace/response_notification/email/body.html

```diff
@@ -0,0 +1,58 @@
+{% load i18n %}
+{% load static %}
+
+{% block preview_text %}
+    {% blocktrans trimmed %}
+        Hi {{ thread_username }}
+    {% endblocktrans %}
+{% endblock %}
+
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            {% blocktrans trimmed %}
+                <h1>
+                    Hi {{ thread_username }},
```

Comment:
Is this how our emails usually start? Or do we use the user's actual name?

Reply:
The template said "name or edX username??" so we might want to double-check with Sylvia.